sklearn/manifold/_t_sne.py

```diff
@@ -990,6 +990,8 @@ def _fit(self, X, skip_num_points=0):
                 svd_solver="randomized",
                 random_state=random_state,
             )
+            # Alays output a numpy array, no matter what is configured globally
+            pca.set_output(transform="default")
```

Comment:
```suggestion
            ).set_output(transform="default")  # only output NumPy array
```

Reply:
I'm not a fan of the "chaining" style because it is easy to miss it. At least I often miss it. This case is a bit weird because we set the output of an internal thing, which I think would surprise most readers. Hence I added a comment to try and explain why this happens and put it on a new line to make sure people don't miss it.

So I am -0.5 on chaining, but not a hill I want to die on ;)

Reply:
I prefer not chaining in this case.